### PR TITLE
meditation page setup

### PR DESCRIPTION
**Pull request by kirustar14**
made the meditation page

### CODE REVIEW

File: client/src/css/meditation.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  }
+  
+  .thank-you {
+    font-size: 28px;
+    color: #ffb347;
+    font-weight: bold;
+    font-family: 'Nunito', sans-serif;
+    padding-top: 30px;
+  }
+
```
Comment (Shuzhengz) [inline client/src/css/meditation.css:132]: hm its rly weird that it thinks you dont have a newline here. is there a space on line 132?

Comment (kirustar14) [reply client/src/css/meditation.css]: okay it should be fixed now i believe

**Review by htluu-ucsd — APPROVED** [review]
Great.

**Review by JingtingL — APPROVED** [review]

File: client/src/index.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import './css/index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import { BrowserRouter } from "react-router-dom";
+import Meditation from './components/pages/meditation'
```
Comment (Shuzhengz) [inline client/src/index.tsx:7]: import is unused

**Pull request merged**